packages/shared/src/components/tooltips/ContextMenuItem.tsx

```diff
@@ -1,14 +1,15 @@
 import { Item, ItemProps } from '@dailydotdev/react-contexify';
 import classNames from 'classnames';
 import Link from 'next/link';
-import React, { ReactElement } from 'react';
+import React, { HTMLAttributeAnchorTarget, ReactElement } from 'react';
 import ConditionalWrapper from '../ConditionalWrapper';
 import { IconProps, IconSize } from '../Icon';
 
 export interface ContextMenuItemProps extends Omit<ItemProps, 'children'> {
   href?: string;
   label: string;
   Icon: React.ComponentType<IconProps>;
+  target?: HTMLAttributeAnchorTarget;
```

Comment:
As the props grow, I think it would be best to have the whole anchor attributes in a single property as we discussed in the past, then just pass along whatever is needed.

Reply:
Added in 030a5a0ccb4cc68bd5fbdb828c440306e8fa5884